subsys/bluetooth/controller/hci/hci.c

```diff
@@ -3341,6 +3344,63 @@ static void le_per_adv_recv_enable(struct net_buf *buf, struct net_buf **evt)
 	ccst = hci_cmd_complete(evt, sizeof(*ccst));
 	ccst->status = status;
 }
+
+#if defined(CONFIG_BT_CTLR_SYNC_PERIODIC_ADV_LIST)
+static void le_add_dev_to_pal(struct net_buf *buf, struct net_buf **evt)
```

Comment:
The `pal` abbreviation was suggested by me, mainly because the abbreviate makes sense, but also since we updated the whitelist (now called filter accept list) to `fal`.

Reply:
Thanks for bringing more context to the topic. In my humble opinion it is a bit confusing. This is another abbreviation that is ok for people with experience in controller programming (are familiar with language/abbreviations used there). It may be intimidating for new people to start contributing to controller because it makes the code reading harder. The learning curve, that is already pretty steep, gets even steeper.

Reply:
Just checked the 5.3 Core spec to see if the periodic advertising list was ever referenced as "PAL" and it isn't. There is another feature called PAL that was removed in 5.3, otherwise we'd definitely need to change it, but not sure now. 

It is not a strict requirement that if we use an acronym for the filter accept list (FAL) that we should do the same for this list, but to me it makes sense to treat those two somewhat the same. 

@ppryga How do you feel about the FAL acronym then?

Reply:
@Thalley my feelings about `FAL` are the same as for `PAL`. Is there a `FAL` acronym mentioned in the BT Core spec?

BTW. If there were a PAL used for something else, even though is was removed in 5.3, people may be confused.

Reply:
From what I can tell, neither FAL or PAL is used in the spec, and both are always written fully.

Reply:
> 
> 
> Thanks for bringing more context to the topic. In my humble opinion it is a bit confusing. This is another abbreviation that is ok for people with experience in controller programming (are familiar with language/abbreviations used there). It may be intimidating for new people to start contributing to controller because it makes the code reading harder. The learning curve, that is already pretty steep, gets even steeper.

As a person with the least knowledge around Zephyr (compared to others at this discussion), I agree with ppryga. It becomes hard to understand, Also I can see in other files, similar acronyms are not used, especially when "app developers" will use the header file (and is not mainly developed for kernel uses).

Reply:
I agree, `FAL` and `PAL` are as bad compare to `GATT` or `EATT` or `L2CAP` or `L2CAP ECFC`.

Reply:
The same comment applies here: https://github.com/zephyrproject-rtos/zephyr/pull/38338#discussion_r719292963